src/modules/pokemons/PokemonList.ts

```diff
@@ -11954,6 +11954,14 @@ export const pokemonList = createPokemonArray(
         'levelType': LevelType.mediumfast,
         'exp': 147,
         'catchRate': 45,
+        'evolutions': [
+            WeatherRestrictedLevelEvolution([WeatherType.Harsh_Sunlight], 'Castform', 'Castform (Sunny)', 1),
+            WeatherRestrictedLevelEvolution([WeatherType.Rain], 'Castform', 'Castform (Rainy)', 1),
+            WeatherRestrictedLevelEvolution([WeatherType.Thunderstorm], 'Castform', 'Castform (Rainy)', 1),
+            WeatherRestrictedLevelEvolution([WeatherType.Snow], 'Castform', 'Castform (Snowy)', 1),
+            WeatherRestrictedLevelEvolution([WeatherType.Hail], 'Castform', 'Castform (Snowy)', 1),
+            WeatherRestrictedLevelEvolution([WeatherType.Blizzard], 'Castform', 'Castform (Snowy)', 1),
```

Comment:
These Castform became 1520 eggsteps, lol.
```suggestion
            WeatherRestrictedLevelEvolution([WeatherType.Harsh_Sunlight], 'Castform', 'Castform (Sunny)', 1, true),
            WeatherRestrictedLevelEvolution([WeatherType.Rain, WeatherType.Thunderstorm], 'Castform', 'Castform (Rainy)', 1, true),
            WeatherRestrictedLevelEvolution([WeatherType.Snow, WeatherType.Hail, WeatherType.Blizzard], 'Castform', 'Castform (Snowy)', 1, true),
```